src/CMakeLists.txt

```diff
@@ -31,10 +31,38 @@
 
 project(vltrace C)
 
+include(CheckCSourceRuns)
+
 add_cstyle(src)
 
 find_package(PkgConfig)
-pkg_check_modules(LIBBCC REQUIRED libbcc)
+
+if(PKG_CONFIG_FOUND)
+	pkg_check_modules(LIBBCC REQUIRED libbcc)
+else()
+	find_package(LIBBCC QUIET REQUIRED)
```

Comment:
QUIET *and* REQUIRED? How is this supposed to work?

Reply:
It is strange, but it works :-) anyway I fixed it.